### PR TITLE
Fixed 19800 Contact us : design improvement

### PR DESCRIPTION
**Pull request by suryakant-krish**
### Fixed Issues (if relevant)

1. magento/magento2#19800: Contact us : design improvement

### Manual testing scenarios (*)

1. Go to frontend > choose to contact us link from footer link.

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @suryakant-krish. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me 2.3-develop instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

File: app/code/Magento/Contact/view/frontend/web/css/source/_module.less [hunk 1]
```diff
@@ -0,0 +1,50 @@
+// /**
```
Comment (VladimirZaets) [inline app/code/Magento/Contact/view/frontend/web/css/source/_module.less]: Hi @suryakant-krish, thanks for collaboration. I think `//` comments tags is redundant

File: app/code/Magento/Contact/view/frontend/web/css/source/_module.less [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+//  * Copyright © Magento, Inc. All rights reserved.
+//  * See COPYING.txt for license details.
+//  */
+
```
Comment (VladimirZaets) [inline app/code/Magento/Contact/view/frontend/web/css/source/_module.less]: Please remove empty lines

Comment (suryakant-krish) [reply app/code/Magento/Contact/view/frontend/web/css/source/_module.less]: Hello VladimirZaets,

I have removed // comment tag.

Comment (DanielRuf) [reply app/code/Magento/Contact/view/frontend/web/css/source/_module.less]: This is not correct, please use /** without any extra // at the beginning.

Also please remove the empty lines.

**Review by DanielRuf — CHANGES_REQUESTED** [review]

**Comment by suryakant-krish** [comment]
Hello DanielRuf,

I have removed extra space and empty lines.

**Review by DanielRuf — CHANGES_REQUESTED** [review]
Still not correct, please remove the leading `//` in line 1 and 2 and keep line 3. And there was a line before line 1.

**Comment by DanielRuf** [comment]
`//` is for single line comments, `/**/` for block comments.

**Comment by suryakant-krish** [comment]
I have updated that.

**Review by DanielRuf — CHANGES_REQUESTED** [review]
Please put the closing `*/` in a new (empty) line.

**Comment by suryakant-krish** [comment]
Added close comment in new empty line.

**Review by DanielRuf — CHANGES_REQUESTED** [review]
First line is still not correct.
Please see https://github.com/suryakant-krish/magento2/blob/2e08f60e1bd43ccf45a89364e76092cbc6d6da4e/app/code/Magento/Theme/view/frontend/web/css/validate.css

**Comment by suryakant-krish** [comment]
I have updated as well as you have provided link. 
https://github.com/suryakant-krish/magento2/blob/2e08f60e1bd43ccf45a89364e76092cbc6d6da4e/app/code/Magento/Theme/view/frontend/web/css/validate.css

**Review by DanielRuf — APPROVED** [review]

**Review by VladimirZaets — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @VladimirZaets, thank you for the review.
[ENGCOM-3763](https://jira.corp.magento.com/browse/ENGCOM-3763) has been created to process this Pull Request

**Comment by magento-engcom-team** [comment]
Hi @suryakant-krish. Thank you for your contribution. 
We will aim to release these changes as part of 2.3.1. 
Please check the release notes for final confirmation.

**Pull request merged**